egs/thchs30/s5/local/dae/add-noise-mod.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
Add a copyright notice and author names, please. This should be probably done for all the other scripts you've written as well.